lib/l10n-en_GB.js

```diff
@@ -53,10 +53,25 @@ export const messages = {
     'headers.h1-title.not-found': 'Cannot find title or h1 of the document.',
     'headers.h1-title.not-match':
         "Content of title and h1 do not match. Text of title is '${titleText}' while h1 is (transformed into) '${h1Text}'.",
+    // headers/shortname
+    'headers.shortname.shortname-lowercase':
+        'The shortname of the document must in lowercase, but `${shortname}` is found.',
+    'headers.shortname.this-latest-shortname':
+        '<em>Shortnames</em> differ between This version (${thisShortname}) and Latest Versions (${latestShortname}).',
+    'headers.shortname.this-previous-shortname':
+        '<em>Shortnames</em> differ between This version (${thisShortname}) and Previous Versions (${previousShortname}).',
+    'headers.shortname.history-syntax':
+        'Wrong syntax for History link, please use the link "https://www.w3.org/standards/history/${shortname}".',
+    'headers.shortname.history-bad-previous':
+        'This document contains a shortname change with previous shortname "${previousShortname}", but the it seems be not correct because "${url}" doesn\'t exist.',
+    'headers.shortname.this-rescinds-shortname':
+        '<em>Shortnames</em> differ between This Version (${thisShortname}) and Rescinds this Recommendation (${rescindsShortname}).',
+    'headers.shortname.shortname-existed':
+        'The ${status} should use an unique shortname.',
```

Comment:
The status will always be 'First Public Working Draft' the way it's being used in `shortname.js`. No need to pass an extra parameter.
```suggestion
        'FPWDs must use a new shortname.',
```